README.md

```diff
@@ -18,6 +18,10 @@ RCOS offers a huge community of open source developers skilled in nearly any pla
 
 RCOS is offered for credit at RPI and fulfills a 4000 level technical elective.
 
+# How to join RCOS
+
+If your an RPI student, hop in on our first meeting of the semester or join [our slack](https://rcos.slack.com) and message a faculty coordinator (@mskmoorthy or @turnew2). If you're looking to get involved with RCOS as a member outside of RPI, email a [coordinator](mailto:richiejoeyoung@gmail.com).
```

Comment:
For the record, I'm fine with having my email hard coded but thank you for considering :p. Having something more official looking like @kelleyb suggested is probably a good idea though